### PR TITLE
temp PR

### PR DESCRIPTION
**Pull request by MillenniumFalconMechanic**
## Reason for Change

- #TICKET_NUMBER
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

- add
- remove
- modify

## Testing steps

- Either list QA steps or reasoning you feel QA is unnecessary
- Describe how you made sure to know that your changes worked. Should allow someone else to go verify your code without in depth knowledge.
- "Unit tested only", "tested in rdev by a, b, c, verifying feature worked by... ", "manually ran pipeline locally with these results: ..."

### CODE REVIEW

File: backend/layers/persistence/persistence_mock.py [hunk 1]
```diff
@@ -267,7 +267,7 @@ def get_collection_version_with_datasets(
         version = self.collections_versions.get(version_id.id)
         if not version:
             return None
-        version = self._update_version_with_canonical(version, update_datasets=True)
+        version = self._update_version_with_canonical(version, update_datasets=True, get_tombstoned=get_tombstoned)
```
Comment (danieljhegeman) [inline backend/layers/persistence/persistence_mock.py:270]: this is a great catch. We should introduce this fix independent of any other concerns.

File: backend/portal/api/portal_api.py [hunk 2]
```diff
@@ -848,6 +850,6 @@ def get_dataset_identifiers(url: str):
         "dataset_id": dataset_id,
         "collection_id": collection_id,
         "collection_visibility": "PUBLIC" if collection.published_at is not None else "PRIVATE",
-        "tombstoned": False,  # No longer applicable
+        "tombstoned": False,  # Required for tombstoned redirect from Explorer
```
Comment (danieljhegeman) [inline backend/portal/api/portal_api.py:853]: want to go ahead and use the tombstone value of the Dataset here?

**Comment by danieljhegeman** [comment]
@MillenniumFalconMechanic this looks like a fine beginning to pursuing the non-error-based route to inform the Explorer that the Dataset is tombstoned, i.e., by returning 200 resp code but with tombstone = true

Anything else you want me to look at?

**Comment by MillenniumFalconMechanic** [comment]
Closing; [Redirect and toast for withdrawn datasets is regressed](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-explorer/648) is currently paused.

**Pull request closed without merging**